packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -172,6 +172,12 @@ class Directionality extends InheritedWidget {
 /// ```
 /// {@end-tool}
 ///
+/// Directly drawing an [Image] with opacity is faster than using [Opacity]
```

Comment:
This section does not detail how one would go about drawing an image with an opacity (as the image widget does not take an opacity). Please add a snippet here detailing how to simulate the same with a color and blend mode.

Reply:
Please see the existing sample code from line 160 to 174. Is that what you're looking for?